packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -146,6 +146,8 @@ export const messages = {
   force_lightning_interface_create_text: 'SFDX: Create Aura Interface',
   force_function_create_text: 'SFDX: Create Function',
   force_function_start_text: 'SFDX: Start Function',
+  force_function_start_no_org_auth:
+    'No default org is set. SFDX: Start Function will continue without org features. Run "SFDX: Set a Default Org" to select an active scratch org, or run "SFDX: Authorize a Dev Hub" and "SFDX: Create a Default Scratch Org" to authenticate to a Dev Hub and create a default scratch org.',
```

Comment:
@ekapner @danyusf also see the screenshot attached to description

Reply:
@xyc - chatting with Dan to understand limitations for Start without an org. Seems like you can start and test locally. And only need an org when you want to test in one. So, Dan and I will provide feedback once I understand what "without org features" really means.

Reply:
If the function uses org features, org auth is needed - For example if it uses `context.org.data.query`. If it doesn't use org features, it's not necessary to auth with an org.

Reply:
`without org features` meaning functions that don't use org features would work fine. For functions that use org features, "SFDX: Start Function" runs fine with the message shown, but it will encounter an error when running "SFDX: Invoke Function" about org auth.

Reply:
No default org is set. We recommend that you select an active scratch org (SFDX: Set a Default Org) or create a new scratch org (SFDX: Authorize a Dev Hub, then SFDX: Create a Default Scratch Org). 

@danyusf - do you think it's obvious that we're telling them to run the commands in parens?

Reply:
And @danyusf - should we write this to be generic enough to stand test of time when sandboxes and prod orgs are supported?

Reply:
@ekapner I think that's fine, and yes, I think a Functions dev will understand the next steps. Regarding making the language work for future support for prod/sandbox orgs, this is tricky (just because there's a lot to do to make things "production ready" and things may get changed, or maybe we'll have a separate workflow for deploying Functions to prod/sandbox orgs). I would say the best we can do for now is let the text reflect the current state of Functions.

Reply:
Thanks @ekapner @danyusf I'll update the text according to your recommendation for now